### PR TITLE
Read graph

### PR DESCRIPTION
**Pull request by Vennik**
Created an graph parser and a simple test.
Needs further extension, but it's a start.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,142 +1,148 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (qurben) [inline pom.xml:2]: Is 4 spaces Intellij standard?

Comment (Vennik) [inline pom.xml:2]: Yes, 4 spaces it Intellij standard, it kept hammering on it.

Comment (qurben) [inline pom.xml:2]: I see, don't know why my IntelliJ didn't complain

**Comment by qurben** [comment]
Looks good :shipit:

**Comment by qurben** [comment]
:shipit:

**Pull request merged**